runtime/src/lib.rs

```diff
@@ -320,6 +321,42 @@ impl pallet_sudo::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 }
 
+// Configure collective pallet for council
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = 3 * MINUTES;
+	pub const CouncilMaxProposals: u32 = 100;
+	pub const CouncilMaxMembers: u32 = 100;
```

Comment:
I don't think we need that many council members initially, or for testing, can we limit it to 3 for now?